assets/app/view/game/round/auction.rb

```diff
@@ -174,7 +174,9 @@ def render_company_actions(company)
                     })
 
           buttons = []
-          if @step.may_bid?(company) && @step.min_bid(company) <= @step.max_place_bid(@current_entity, company)
+          if @step.may_bid?(company) && @step.min_bid(company) <= @step.max_place_bid(@current_entity, company) &&
+            (!@step.respond_to?(:can_bid_company?) || @step.can_bid_company?(@current_entity, company))
```

Comment:
Why are these checks needed now in the common view code? Why didn't 22 need it?

Reply:
Removed - this is checked in may_bid?(company) now